src/Bicep.LangServer.IntegrationTests/HoverTests.cs

```diff
@@ -69,7 +69,9 @@ public async Task HoveringOverSymbolReferencesAndDeclarationsShouldProduceHovers
 
                 switch (symbol!.Kind)
                 {
-                    case SymbolKind.Error:
+                    // when a namespace does not exist, instance function call contains a null hover range
```

Comment:
I'll update this comment to be a little bit more explicit, it should say:

"When a namespace value is not found..."